doc/dox_comments/header_files/aes.h

```diff
@@ -529,6 +545,7 @@ int wc_GmacUpdate(Gmac* gmac, const byte* iv, word32 ivSz,
     Aes enc;
     key[] = { some 16, 24, or 32 byte length key };
 
+    wc_AesInit(&aes, HEAP_HINT, INVALID_DEVID); // Make sure devId updated
```

Comment:
Is the `// Make sure devId updated` needed?

Reply:
Yes.  Its in other places. If it belongs there, it belongs here, IMHO.